backend/src/flotilla/api/events_api.py

```diff
@@ -62,11 +63,24 @@ async def post_event(
 ) -> Event:
     """Add a new event to the robot schedule"""
     try:
+        end_time = event_request.start_time + DEFAULT_EVENT_DURATION
```

Comment:
Will we always use the default duration? Maybe the user should have an event_duration optional filed to this post endpoint so they can define it themselves?

Reply:
Currently a default value is enough in my opinion. I don't think the user should be able to set the event_duration, but rather be based on the number of tags and/or based on data from previous missions with the same id. Can make a card for setting the event_duration based on number of tags.